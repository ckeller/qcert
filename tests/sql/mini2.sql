select
	l_returnflag,
  count(*) as count
from
	lineitem
where
	l_shipdate <= date '1993-12-01'
group by
	l_returnflag;