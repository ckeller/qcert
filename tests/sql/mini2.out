[{"count" : {"$nat":2}, "l_returnflag" : "R"},
 {"count" : {"$nat":2}, "l_returnflag" : "A"}]